### PR TITLE
[Part 2] Update the MFA recommendation rollout post based on feedback

### PR DESCRIPTION
**Pull request by bettymakes**
Contributes to [Phase 2 - Recommending MFA on Most Downloaded Gems](https://github.com/Shopify/ruby-conventions/milestone/3)

## 🤷‍♀️ What problem are you solving?
Aditya's provided us with another round of feedback. It seems like we're close now 🤞.

## 📋 How will you solve this?
The following are items to be addressed:

### Feedback Items
- [x] 1️⃣ [Feedback](https://github.com/rubygems/rubygems.github.io/pull/110#discussion_r889688115): Clarify who is being affected and note the email remainders and its expected cadence 
  ```
  Lets also talk about email reminders to clarify who is affected. Something along the lines:
  
  If you are owner of any of these gems, you will receive an email reminder to enable mfa one 
  month and one week before this goes into effect.
  ```
- [x] 2️⃣ [Feedback](https://github.com/rubygems/rubygems.github.io/pull/110#discussion_r889688115): Clarify MFA authentication levels and note our recommendation
  ```
  We should clarify that MFA requirement means ui_and_gem_signin only. Link the guide to 
  clarify what the level means. We can also add that we recommend enableing ui_and_api 
  level if possible.
  ```
- [x] 3️⃣ [Feedback](https://github.com/rubygems/rubygems.github.io/pull/110#discussion_r889688371): Provide better insight into future plans and offer mediums for giving feedback 
  ```
  It is up to you but if you want, you and mention that we plan to increase the MFA coverage 
  by extending this requirement to more gems in future. If the reader has opinions about this, 
  they can reach us in slack or github issues.
  ```
- [x] 4️⃣ [Feedback](https://github.com/rubygems/rubygems.github.io/pull/110#issuecomment-1146797365): Add doodle of MFA Gemmy [will need to optimize image for web]
  ```
  > "Happy to include the doodle in the blog (and/or email) if we're open to adding some extra 
    sparkles flare sparkles to the blog post."
  
  Please feel free.
  ```

### Other Items
- [x] 🤔 Self-nit: Should we rename the file to something else? The URL will end up being ...
  `https://blog.rubygems.org/2022/06/13/mfa-phase-2-rollout.html`
  "MFA Phase 2 Rollout" is not very descriptive. What's "phase 2"? It'd be better if the URL was more expressive. I _think_ it'd benefit SEO as well based off of the rudimentary SEO knowledge I have lol.

- [x] If we don't rename the file as suggested in the point above, we will still need to update the file name to `mfa-phase-2-rollout`. It currently says `1`. My bad!

### CODE REVIEW

**Comment by bettymakes** [comment]
👋 @tomstuart - Could you give me a hand word-smithing this one? I'm struggling to come up with a way to eloquently incorporate [Feedback Item #3](https://github.com/rubygems/rubygems.github.io/pull/110#discussion_r889688371) (see PR description) into the concluding paragraph. SOS!

**Comment by tomstuart** [comment]
I did some minor tinkering with the opening paragraph as a displacement activity, then rolled my eyes and reluctantly did the thing you actually asked for, muttering as I went.

I also renamed the file to give the post a new slug per your self-nit, so again [this view](https://github.com/Shopify/rubygems.github.io/pull/2/files/e686e11e9809786d5537b4e5430183449486bd3b..bb613a50672ad3c7424693d2c8f94dd530a6e995) might make my content changes easier to review.

**Comment by tomstuart** [comment]
(Be warned that I also had to rebase the branch on top of `mfa-phase-1-rollout` to remove merge conflicts, so you’ll need to do some local Git shenanigans to bring your repo up to date.)

File: _posts/2022-06-13-making-packages-more-secure.md [hunk 1]
```diff
@@ -15,4 +18,8 @@ Starting today (June 13, 2022), the maintainers of at least the top 100 RubyGems
 
 Although this is currently just a recommendation, we will begin to enforce MFA for maintainers of those gems in two months’ time (August 15, 2022). This policy will bring us in line with other package ecosystems (e.g. [npm](https://github.blog/2022-02-01-top-100-npm-package-maintainers-require-2fa-additional-security/)) as well as [GitHub](https://github.blog/2022-05-04-software-security-starts-with-the-developer-securing-developer-accounts-with-2fa/).
 
+Maintainers affected by this policy will receive email reminders to enable MFA one-month and one-week before the enforcement goes into effect. We recommend that maintainers set their MFA [authentication level](https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels) to `UI and API`. However, `UI and gem signin` is acceptable too.
```
Comment (jchestershopify) [inline _posts/2022-06-13-making-packages-more-secure.md]: I think "one month" and "one week".

File: _posts/2022-06-13-making-packages-more-secure.md [hunk 2]
```diff
@@ -4,8 +4,11 @@ layout: post
 author: Betty Li
 author_email: betty.li@shopify.com
 ---
+<p align="center">
+    <img src="/images/gem-with-mfa-flag.png" alt="Doodle of a RubyGem wearing an MFA hat, holding a flag with a checkmark" width="300"/>
```
Comment (jchestershopify) [inline _posts/2022-06-13-making-packages-more-secure.md]: Thank you for including an alt.

**Review by jchestershopify — APPROVED** [review]
One nit, otherwise LGTM.

**Review by aellispierce — APPROVED** [review]
Looks good to me! 🎉

File: _posts/2022-06-13-making-packages-more-secure.md (hunk 1 above)
Comment (tomstuart) [inline _posts/2022-06-13-making-packages-more-secure.md]: Agreed! I also think we need to clarify that these are two separate emails, e.g.:

```suggestion
Maintainers affected by this policy will receive email reminders to enable MFA one month before, and again one week before, the enforcement goes into effect. We recommend that maintainers set their MFA [authentication level](https://guides.rubygems.org/setting-up-multifactor-authentication/#authentication-levels) to `UI and API`. However, `UI and gem signin` is acceptable too.
```

**Review by tomstuart — APPROVED** [review]
month/week nit aside, let’s go! 🥳

**Review by jenshenny — APPROVED** [review]
LGTM, thanks Betty! I also tophatted it to make sure gemmy was looking nice ✨
<details><summary>🎩</summary>
Screen Shot 2022-06-06 at 4 43 37 PM
</details>

File: _posts/2022-06-13-making-packages-more-secure.md (hunk 1 above)
Comment (bettymakes) [reply _posts/2022-06-13-making-packages-more-secure.md]: Ah! You're right! I was in the mind space thinking "one-month" & "one-week" should be compound adjectives (e.g. `one-month notice`). But in this case, `one` is the adjective for `month`. 
No hyphens needed! 🙈

I'll commit the [suggestions](https://github.com/Shopify/rubygems.github.io/pull/2#discussion_r890490349) @tomstuart made in a following comment which address this.

Comment (bettymakes) [reply _posts/2022-06-13-making-packages-more-secure.md]: ✅ Donezo. Thanks Tom!

**Comment by bettymakes** [comment]
I noticed in @jenshenny's  🎩 that the white outline around the MFA Gemmy was missing (since the background of the blog is also white). So I revised the MFA Gemmy to include a drop shadow.

<p align="center">
  <kbd>
    MFA Gemmy with drop shadow on RubyGems Blog
  </kbd>
</p>

**Comment by bettymakes** [comment]
All feedback addressed! Let's goooo 🚀! (Merging back to [our main PR](https://github.com/rubygems/rubygems.github.io/pull/110))

**Pull request merged**